examples/build_symmetry_mirror.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 '''
-This example shows usage of symmetry operation tools for FCC lattice.
+This example shows usage of symmetry operation utils for FCC lattice.
```

Comment:
not sure, about this change since utility functions reside in the `utils` folder